integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -0,0 +1,94 @@
+const _ = require('lodash/fp')
+const fetch = require('node-fetch')
+const pRetry = require('p-retry')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, delay, dismissNotifications, findElement, findText, fillIn, input, select, signIntoTerra, waitForNoSpinners, findInGrid, navChild } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const snapshotName = 'testsnapshot'
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
+
+const withDataRepoCheck = test => async options => {
```

Comment:
Do you mean we shouldn't check this?

Reply:
I think we should consider that, yes. Right now this encodes the idea that "It's ok if data repo is down". Regardless of whether that's the right tactical decision at this point, I think it's good to remember that decisions like this can have a subtle influence of our expectations for our systems.

Reply:
Consider this a TOL, feel free to make the right tactical decision on this.

Reply:
I'm going to leave this as is for now. I think the question of "what the UI tests should do if urls we depend on don't work" is a broader question that I will leave to grown-up UI folks.